### PR TITLE
Moebius faction items

### PR DESCRIPTION
**Pull request by DarkLevelSS13**
## About The Pull Request

- Von-Krabin Stimulator.  Just like NT obelisk, increases stats of people who visit it, it boosts BIO, COG and MEC by +5, and it’s located near RnD machines.
image

- Molitor-Riedel Enricher. Produces universal donor blood if you inject it with nutrients, outputs it in packet ready to use.
image

## Why It's Good For The Game
Adding new steal objectives for traitors, and adding unique items for fraction Moebius.

## Changelog
:cl: DarkLevel
add: Faction item/Von-Krabin Stimulator
add: Faction item/Molitor-Riedel Enricher
fix: biosyphon can now spawn item while in hand
/:cl:

:cl: INFRARED_BARON
imageadd: Von-Krabin Stimulator sprites
imageadd: Molitor-Riedel Enricher sprites
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/von-krabin.dm [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/obj/item/device/von_krabin
+	name = "Von-Krabin Stimulator"
+	desc = "The Von-Krabin Stimulator."
```
Comment (MadmanMartian) [inline code/game/objects/items/devices/von-krabin.dm:3]: Give a brief description of what it does at least.

File: code/game/objects/items/devices/von-krabin.dm [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	update_icon()
+
+/obj/item/device/von_krabin/proc/check_for_faithful(list/affected)
+	var/got_follower = FALSE
+	var/list/mob/living/carbon/human/no_longer_affected = currently_affected - affected
+	currently_affected -= no_longer_affected
+	for(var/mob/living/carbon/human/H in no_longer_affected)
+		for(var/stat in stats_buff)
+			H.stats.removeTempStat(stat, "von_krabin")
+			to_chat(H, SPAN_NOTICE("Your knowledge of [stat] slightly decreese."))
```
Comment (MadmanMartian) [inline code/game/objects/items/devices/von-krabin.dm:54]: "your knowledge of [stat] slightly decreases once you leave the von krabin's influence."

File: code/modules/reagents/enricher.dm [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+			else
+				reagent_amout = reagent.volume
+				reagent.remove_self(reagent_amout) //Purge useless reagents out
+
+		if(blood_amout)
+			while(blood_amout)
+				if(blood_amout > 200)
+					var/obj/item/weapon/reagent_containers/blood/OMinus/blood_pack = new /obj/item/weapon/reagent_containers/blood/OMinus(get_turf(src))
+					blood_amout -= 200
+					visible_message(SPAN_NOTICE("[name] drop [blood_pack]."))
```
Comment (MadmanMartian) [inline code/modules/reagents/enricher.dm:36]: Reads like debug text. Improve or remove.

File: code/modules/reagents/enricher.dm [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+				else
+					var/obj/item/weapon/reagent_containers/blood/empty/blood_pack = new /obj/item/weapon/reagent_containers/blood/empty(get_turf(src))
+					blood_pack.reagents.add_reagent("blood", blood_amout, list("donor"=null,"viruses"=null,"blood_DNA"=null,"blood_type"="O-","resistances"=null,"trace_chem"=null))
+					blood_amout = 0
+					visible_message(SPAN_NOTICE("[name] drop [blood_pack]."))
```
Comment (MadmanMartian) [inline code/modules/reagents/enricher.dm:41]: Reads like debug text. Improve or remove.

File: code/modules/reagents/enricher.dm [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+		else
+			visible_message("[name] beeps, \"Not enough nutriment to produce blood\"")
```
Comment (MadmanMartian) [inline code/modules/reagents/enricher.dm:43]: "\The [src] beeps, \"Not enough nutriment to produce blood.\"."

File: code/modules/reagents/enricher.dm [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+	else
+		visible_message("[name] beeps, \"Don't have any reagents to produce blood\"")
```
Comment (MadmanMartian) [inline code/modules/reagents/enricher.dm:45]: "\The [src] beeps, \"Insufficient reagents to produce blood.\"."

File: code/game/objects/items/devices/von-krabin.dm (hunk 1 above)
Comment (DarkLevelSS13) [reply code/game/objects/items/devices/von-krabin.dm]: Fixed.

File: code/game/objects/items/devices/von-krabin.dm (hunk 2 above)
Comment (DarkLevelSS13) [reply code/game/objects/items/devices/von-krabin.dm]: Fixed.

File: code/modules/reagents/enricher.dm (hunk 3 above)
Comment (DarkLevelSS13) [reply code/modules/reagents/enricher.dm]: Fixed.

File: code/modules/reagents/enricher.dm (hunk 4 above)
Comment (DarkLevelSS13) [reply code/modules/reagents/enricher.dm]: Fixed.

File: code/modules/reagents/enricher.dm (hunk 5 above)
Comment (DarkLevelSS13) [reply code/modules/reagents/enricher.dm]: Fixed.

File: code/modules/reagents/enricher.dm (hunk 6 above)
Comment (DarkLevelSS13) [reply code/modules/reagents/enricher.dm]: Fixed.

**Pull request merged**